scripts/Tools/Makefile

```diff
@@ -898,15 +899,15 @@ $(PIOLIB) : $(MPISERIAL) $(GPTLLIB)
 
 $(CSMSHARELIB):  $(MCTLIBS) $(PIOLIB) $(GPTLLIB)
 
-ifneq ($(MODEL),csm_share)
+ifneq ($(findstring csm_share,$(MODEL)),csm_share)
   $(OBJS):  $(CSMSHARELIB)
 else
   complib: install_lib
 endif
 
 install_lib: $(COMPLIB)
 	$(CP) -p $(COMPLIB) $(CSMSHARELIB)
-	$(CP) -p *.$(MOD_SUFFIX) *.h $(INCLUDE_DIR)
+	$(CP) -p *.$(MOD_SUFFIX)  $(INCLUDE_DIR)
```

Comment:
Why remove *.h?   I think that it may be important for some compsets or compilers.

Reply:
When building the library for the CESM_share repository, there are no *.h.  Which causes an error.  Let me see what I can 
do about this.